app/assets/javascripts/application.js

```diff
@@ -12,5 +12,6 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require bootstrap-sprockets
 //= require jasny-bootstrap
```

Comment:
Jasny bootstrap is part of the old system and adds on some additional bootstrap components. Do we want to use that library?